keymap/sublime.js

```diff
@@ -409,6 +409,24 @@
 
   map[cK + ctrl + "Backspace"] = "delLineLeft";
 
+  map["Backspace"] = function(cm) {
+    if (cm.somethingSelected()) {
+      return CodeMirror.Pass;
+    }
+
+    var tabSize = cm.getOption('tabSize');
+    var spaces = new Array(tabSize).join(' ');
+    var cursor = cm.getCursor();
+    var toStartOfLine = cm.getRange({line: cursor.line, ch: 0}, cursor)
+      .replace("\t", spaces);
```

Comment:
You should be able to use `CodeMirror.countColumn(cm.getLine(cursor.line), cursor.ch, tabSize)` to get the column at which the cursor is. That's probably cleaner.